stable/vpa/values.yaml

```diff
@@ -13,9 +13,23 @@ fullnameOverride: ""
 # podLabels -- Labels to add to all pods
 podLabels: {}
 rbac:
-  # rbac.create -- If true, then rbac resources (clusterroles and clusterrolebindings) will be created for the selected components.
+  # rbac.create -- If true, then rbac resources (ClusterRoles and ClusterRoleBindings) will be created for the selected components.
   # Temporary rbac resources will still be created, to ensure a functioning installation process
   create: true
+  # rbac.clusterRoles -- Extra rbac rules for Clusterroles
+  clusterRoles:
```

Comment:
I would prefer to use a more descriptive name, such as `additionalClusterRoles`

Reply:
I think your proposal can confuse a user, because `additionalClusterRoles` can be interpreted as new ClusterRoles, but in this case they already exist.

Reply:
to me `rbac.clusterRoles` indicates that you're setting _all_ the clusterroles, not adding additional ones. Do you have a different name in mind that might be more descriptive and avoid the confusion you suggest?

Reply:
It can be the same as in `goldilocks` - [`extraRules`](https://github.com/FairwindsOps/charts/blob/master/stable/goldilocks/values.yaml#L40-L41). What do you think?

Reply:
Another idea in my head - `additionalClusterRoleRules`. It's a bit long, but as clear as possible (:

Reply:
Either of those is fine for me. I like being consistent with the Goldilocks chart.

Reply:
I've renamed `clusterRoles` value to `extraRules`.